src/nvim/ex_docmd.c

```diff
@@ -3219,12 +3222,12 @@ char *skip_range(const char *cmd, int *ctx)
   return (char *)cmd;
 }
 
-static void addr_error(cmd_addr_T addr_type)
+static void addr_error(cmd_addr_T addr_type, char **errormsg)
```

Comment:
I think it would be better to return an error instead?

```suggestion
static const char *addr_error(cmd_addr_T addr_type)
```